app/R/plot_beds.R

```diff
@@ -0,0 +1,40 @@
+#' Function to plot outputs of predict_beds
+#'
+#' This function provides custom plots for the output of `predict_beds`.
+#'
+#' @param x the output of `predict_beds`
+#'
+#' @param ... further arguments passed to `plot.projections`
+#' 
+#' @author Thibaut Jombart
+#' 
+#' @examples
+#' 
+#' ## get forecast for admissions
+#' x <- predict_admissions(Sys.Date(),
+#'                         n_start = 40,
+#'                         doubling = 5,
+#'                         doubling_error = 1,
+#'                         duration = 14) 
+#' x
+#' 
+#' ## make toy duration of hospitalisation (exponential distribution)
+#' r_duration <- function(n = 1) rexp(n, .2)
+#'
+#' ## get daily bed needs predictions
+#' beds <- predict_beds(x$date, x$mean, r_duration)
+#' beds
+#' plot_beds(beds)
+
+plot_beds <- function(x, ...) {
+  plot(x,
+       quantiles = c(.025, .5),
```

Comment:
Do you mean to have 0.025 and 0.975 quantiles here?

Reply:
@thibautjombart I'm having trouble with R and package updates - happy to approve reply if this is holding things up - where/how do I do that? Thanks

Reply:
@esnightingale I had a similar question but digging around it seems that the plot method generates a full list of quantiles to be requested by taking `quantiles` and forming `1-quantiles`.